
Updated version of an earlier patch to make stunnel default to the
'authpriv' syslog facility, instead of daemon.

diff -ur stunnel-5.60.orig/doc/stunnel.8.in stunnel-5.60/doc/stunnel.8.in
--- stunnel-5.60.orig/doc/stunnel.8.in	2021-05-30 15:19:48.000000000 +0000
+++ stunnel-5.60/doc/stunnel.8.in	2021-11-19 00:11:36.690601036 +0000
@@ -204,7 +204,7 @@
 all levels numerically less than it will be shown.  Use \fIdebug = debug\fR or
 \&\fIdebug = 7\fR for greatest debugging output.  The default is notice (5).
 .Sp
-The syslog facility 'daemon' will be used unless a facility name is supplied.
+The syslog facility 'authpriv' will be used unless a facility name is supplied.
 (Facilities are not supported on Win32.)
 .Sp
 Case is ignored for both facilities and levels.
diff -ur stunnel-5.60.orig/doc/stunnel.html.in stunnel-5.60/doc/stunnel.html.in
--- stunnel-5.60.orig/doc/stunnel.html.in	2021-05-30 15:19:48.000000000 +0000
+++ stunnel-5.60/doc/stunnel.html.in	2021-11-19 00:12:08.301695748 +0000
@@ -244,7 +244,7 @@
 
 <p>Level is one of the syslog level names or numbers emerg (0), alert (1), crit (2), err (3), warning (4), notice (5), info (6), or debug (7). All logs for the specified level and all levels numerically less than it will be shown. Use <i>debug = debug</i> or <i>debug = 7</i> for greatest debugging output. The default is notice (5).</p>
 
-<p>The syslog facility &#39;daemon&#39; will be used unless a facility name is supplied. (Facilities are not supported on Win32.)</p>
+<p>The syslog facility &#39;authpriv&#39; will be used unless a facility name is supplied. (Facilities are not supported on Win32.)</p>
 
 <p>Case is ignored for both facilities and levels.</p>
 
diff -ur stunnel-5.60.orig/src/options.c stunnel-5.60/src/options.c
--- stunnel-5.60.orig/src/options.c	2021-08-05 02:19:52.000000000 +0000
+++ stunnel-5.60/src/options.c	2021-11-19 00:16:58.778393825 +0000
@@ -1865,7 +1865,9 @@
     switch(cmd) {
     case CMD_SET_DEFAULTS:
         section->log_level=LOG_NOTICE;
-#if !defined (USE_WIN32) && !defined (__vms)
+#if defined(__sun) && defined(__SVR4) && defined(LOG_AUTHPRIV)
+        new_global_options.log_facility=LOG_AUTHPRIV;
+#elif !defined (USE_WIN32) && !defined (__vms)
         new_global_options.log_facility=LOG_DAEMON;
 #endif
         break;
@@ -1881,14 +1883,18 @@
     case CMD_INITIALIZE:
         break;
     case CMD_PRINT_DEFAULTS:
-#if !defined (USE_WIN32) && !defined (__vms)
+#if defined(__sun) && defined(__SVR4) && defined(LOG_AUTHPRIV)
+        s_log(LOG_NOTICE, "%-22s = %s", "debug", "authpriv.notice");
+#elif !defined (USE_WIN32) && !defined (__vms)
         s_log(LOG_NOTICE, "%-22s = %s", "debug", "daemon.notice");
 #else
         s_log(LOG_NOTICE, "%-22s = %s", "debug", "notice");
 #endif
         break;
     case CMD_PRINT_HELP:
-#if !defined (USE_WIN32) && !defined (__vms)
+#if defined(__sun) && defined(__SVR4) && defined(LOG_AUTHPRIV)
+        s_log(LOG_NOTICE, "%-22s = [facility].level (e.g. authpriv.info)", "debug");
+#elif !defined (USE_WIN32) && !defined (__vms)
         s_log(LOG_NOTICE, "%-22s = [facility].level (e.g. daemon.info)", "debug");
 #else
         s_log(LOG_NOTICE, "%-22s = level (e.g. info)", "debug");
